src/main/kotlin/org/hyacinthbots/lilybot/extensions/util/Reminders.kt

```diff
@@ -424,6 +471,36 @@ class Reminders : Extension() {
 		}
 	}
 
+	/**
+	 * Edits a reminder message to contain cancelled and removes it from the database.
+	 *
+	 * @param reminder The reminder to edit and remove
+	 * @param userId The user to remove the reminder from. Will use interaction author if null
+	 * @since 4.0.0
+	 * @author NoComment1105
+	 */
+	private suspend inline fun EphemeralSlashCommandContext<*>.editAndRemoveReminder(
+		reminder: RemindMeData,
+		userId: Snowflake? = null
+	) {
+		val messageId = Snowflake(reminder.originalMessageUrl.split("/")[6])
+		val message = event.kord.getGuild(reminder.guildId)!!
+			.getChannelOf<GuildMessageChannel>(reminder.channelId)
+			.getMessageOrNull(messageId)
+
+		completeOrCancelReminder(message, true)
+
+		RemindMeCollection().removeReminder(guild!!.id, userId ?: user.id, reminder.id)
+	}
+
+	/**
+	 * Gets a reminders information and produces a formatted string to send in an embed.
+	 *
+	 * @param reminder The reminder to get the information from
+	 * @return A formatted string to send in an embed
+	 * @since 3.5.2
```

Comment:
wrong version

Reply:
No that's an old function that wasn't doc'd. This is when it was added